src/toil_scripts/tools/aligners.py

```diff
@@ -56,3 +56,79 @@ def run_star(job, cores, r1_id, r2_id, star_index_url):
     transcriptome_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'rnaAligned.toTranscriptome.out.bam'))
     sorted_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'rnaAligned.sortedByCoord.out.bam'))
     return transcriptome_id, sorted_id
+
+
+def run_bwakit(job, threads, r1, r2, ref, fai, amb, ann, bwt, pac, sa, uuid, library, platform, program_unit,
+               alt=None, rg_line=None, sort=True, trim=False):
+    """
+    Runs BWA-Kit to align a fastq file or fastq pair into a BAM file
+
+    :param JobFunctionWrappingJob job: Passed by Toil automatically
+    :param int threads: Number of threads to use
+    :param str r1: FileStoreID of first fastq
+    :param str r2: FileStoreID of second fastq (None if not using paired data)
+    :param str ref: FileStoreID of reference
+    :param str fai: FileStoreID of fai
+    :param str amb: FileStoreID of amb
+    :param str ann: FileStoreID of ann
+    :param str bwt: FileStoreID of bwt
+    :param str pac: FileStoreID of pac
+    :param str sa: FileStoreID of sa
+    :param str uuid: UUID of sample (for readgroup)
+    :param str library: library of sample (for readgroup)
+    :param str platform: platform of sample (for readgroup)
+    :param str program_unit: program unit for sample (for readgroup)
+    :param str alt: FileStoreID for alt sequence
+    :param str rg_line: rg line. Use the format "@RG\\tID:foo\\tLB:bar" ...
```

Comment:
That format line isn't really descriptive. I'd go more for something like 

> BAM read group header line (`@RG`), as defined [on page 3 of the SAM spec](https://samtools.github.io/hts-specs/SAMv1.pdf). Tabs should be escaped, e.g., `@RG\\tID:foo\\tLB:bar...` for the read group "foo" from sequencing library "bar". Multiple `@RG` lines can be defined, but should be split by an escaped newline `\\n`, e.g., `@RG\\tID:foo\\t:LB:bar\\n@RG\\tID:santa\\tLB:cruz`.

Yes, I know it's verbose, but I lost several hours of my life several months ago to this `@RG` header business, and good documentation could save someone else that hassle.